VAMobile/src/components/FormWrapper/FormFields/Picker/VAModalPicker.test.tsx

```diff
@@ -84,21 +80,21 @@ context('VAModalPicker', () => {
   describe('when an option is selected', () => {
     it('should update selected to the value of that option and select done', async () => {
       await waitFor(() => {
-        selectionButtons[0].props.onPress()
+        fireEvent.press(component.getByTestId('Java'))
       })
 
       await waitFor(() => {
-        doneButton.props.onPress()
+        fireEvent.press(doneButton)
       })
 
       expect(selected).toEqual('java')
     })
 
     it('should not update selected to the value of that option and select cancel', async () => {
       await waitFor(() => {
-        selectionButtons[0].props.onPress()
+        fireEvent.press(component.getByTestId('Java'))
 
-        cancelButton.props.onPress()
+        fireEvent.press(cancelButton)
```

Comment:
replace with fireEvent.press(screen.getByRole('button, { name: 'Cancel' })) and remove the cancel button usage

Reply:
Updated.